src/EnergyPlus/Autosizing/Base.cc

```diff
@@ -141,20 +179,16 @@ namespace EnergyPlus {
                                       Optional_string_const UsrDesc,
                                       Optional<Real64 const> UsrValue) {
 
-        static bool MyOneTimeFlag(true);
-
-        // Formats
+        static constexpr auto Format_990("! <Component Sizing Information>, Component Type, Component Name, Input Field Description, Value\n");
         static constexpr auto Format_991(" Component Sizing Information, {}, {}, {}, {:.5R}\n");
 
         // to do, make this a parameter. Unfortunately this function is used in MANY
         // places so it involves touching most of E+
         auto &outputFiles = EnergyPlus::OutputFiles::getSingleton();
-        if (MyOneTimeFlag) {
-            static constexpr auto Format_990(
-                    "! <Component Sizing Information>, Component Type, Component Name, Input Field Description, Value\n");
+        if (oneTimeBaseSizerFlag) {
             if (ReportSizingManager::MyOneTimeFlag) print(outputFiles.eio, Format_990);
```

Comment:
Is this right?  That we still need to also check the one time flag out in the report sizing manager?

Reply:
Yes, that prints the header for component sizing. We only want 1 of those. When this is complete we can remove that.